Code/Common/include/sitkImage.h

```diff
@@ -149,6 +149,17 @@ namespace simple
      * number of components for each pixel is returned.
      */
     unsigned int GetNumberOfComponentsPerPixel( void ) const;
+    /** \brief Get the number of pixels in each image
+     *
+     *  Some pixels may have more than one component, so
+     *  GetNumberOfPixels should be multiplied by
+     *  GetNumberOfComponentsPerPixel in order to calculate the
+     *  the total number of values stored congituously for the image.
+     *
+     * The GetNumberOfPixels produces the same result
+     * as GetWidth()*GetHeight()*GetDepth().  This value
```

Comment:
The above is incorrect. For 2D images as GetDepth() is 0, and SimpleITK also has the option to support 4D images too. And the last sentence appears to be unfinished.